lib/perl/Genome/Model/Tools/Vcf/EvaluateVcf.pm

```diff
@@ -0,0 +1,790 @@
+
+    my $tn = $self->stat_true_negatives();
+    if ($tn == 0) {
+        return 0;
+    }
+
+    my $stat =
+      ($results->{true_negatives} - $results->{false_positive_partial}) / $tn;
+    return $stat;
+}
+
+sub stat_exact_ppv {
+    my $self    = shift;
+
+    my $fp_exact = $self->stat_false_positive_exact();
+    my $tp_exact = $self->stat_true_positive_found_exact();
+
+    my $denominator = $fp_exact + $tp_exact;
+    if ($denominator == 0) {
+        return 0;
+    }
+
+    my $stat = $tp_exact / $denominator;
+    return $stat;
+}
+
+sub stat_partial_ppv {
+    my $self    = shift;
+
+    my $fp_partial = $self->stat_false_positive_partial();
+    my $tp_partial = $self->stat_true_positive_found_partial();
+
+    my $denominator = $fp_partial + $tp_partial;
+    if ($denominator == 0) {
+        return 0;
+    }
+
+    my $stat = $tp_partial / $denominator;
+    return $stat;
+}
+
+sub stat_vcf_lines_overlapping_tn {
+    my $self    = shift;
+    my $results = $self->get_rawstats();
+    my $stat    = $results->{false_positives_in_roi};
+    return $stat;
+}
+
+sub stat_lines_specificity_in_tn_only {
+    my $self    = shift;
+    my $results = $self->get_rawstats();
+
+    # this is more accurate than the other specificity measures, but
+    # doesn't take partial into account
+
+    my $tn = $self->stat_true_negatives();
+    if ($tn == 0) {
+        return 0;
+    }
+
+    my $stat = ($tn - $results->{false_positives_in_roi}) / $tn;
+    return $stat;
+}
+
+sub help_brief {
+    return "Compare or Validate VCF files to a Gold Standard";
+}
+
+sub help_detail {
+    my $doc = q{
+
+    BACKGROUND
+    ==========
+
+    For validation, it is important to compare calls from the pipeline +
+    reporting framework and calculate sensitivity, specificity and positive
+    predictive value of the pipeline's calls versus the gold standard.
+    The standard currency for doing this is a VCF file. A procedure was
+    developed for simplifying callsets and then comparing them to each
+    other. It roughly follows the recommendations of NIST as detailed in the
+    [Genome In a Bottle paper][1].
+
+    METHODOLOGY
+    ===========
+
+    This tool compares a gold-standard VCF to an experimental VCF and a BED
+    file of true negative positions.
+
+    Genotype comparisons are done using [joinx vcf-compare][2]. This program
+    reports an exact and a partial genotype match to the gold standard VCF.
+    Exact matches are straightforward, if both samples report the variant than
+    it is an exact match. Partial matches are more complex, joinx reports the
+    number of unique, matching alternate alleles at each site giving a zygosity
+    independent measure of concordance. For example, a heterozygous variant in
+    the gold sample and a homozygous variant in the evaluation sample would
+    yield a single partial match.
+
+    In order to calculate metrics, the following calculations were used and are
```

Comment:
It might be better to use `are` instead of `were` unless specifically referring to the time of implementation rather than the time of running.